src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp

```diff
@@ -134,7 +134,7 @@ namespace {
         this->updateNetwork(ext_network, handlerContext.currentStep);
     }
 
-    void Schedule::handleCOMPDAT(const HandlerContext& handlerContext, const ParseContext& parseContext, ErrorGuard& errors) {
+    void Schedule::handleCOMPDAT  (const HandlerContext& handlerContext, const ParseContext& parseContext, ErrorGuard& errors) {
```

Comment:
this is the only commit that i looked at ("tabulated" made me curious.)

I have never seen different amounts of whitespace between function name and the parenthesis. Why would I want that?

Reply:
Try `grep 'void Schedule::handle' src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp`.

Reply:
I admit it looks better. However, the convention that the parenthesis immediately follows a function name is well established in the code, so I would wish that could be kept.

Reply:
> I admit it looks better. However, the convention that the parenthesis immediately follows a function name is well established in the code, so I would wish that could be kept.

Besides, if we ever get started on our [ClangFormat](https://github.com/OPM/opm-grid/pull/386) experiment in earnest, all of this discussion is moot.